src/App.js

```diff
@@ -46,4 +49,4 @@ function App() {
     );
 }
 
-export default App;
+export default withWrapper(App);
```

Comment:
do we need withWrapper ?

Reply:
Not really, since we implemented the spacing HOC for learning purposes. So we are stilling using it.